src/main/scala/edu/ie3/simona/config/ConfigFailFast.scala

```diff
@@ -203,24 +205,37 @@ case object ConfigFailFast extends LazyLogging {
     * @param timeConfig
     *   the time config
     */
-  private def checkDateTime(
+  private def checkTimeConfig(
       timeConfig: SimonaConfig.Simona.Time
   ): Unit = {
 
-    // check if the provided date/time values match the SimonaConstants definition for date/time
-    val timeAndDates = Map(
-      "simonaConfig.simona.time.startDateTime" -> timeConfig.startDateTime,
-      "simonaConfig.simona.time.endDateTime" -> timeConfig.endDateTime
-    )
-    timeAndDates.foreach { case (configParameter, dateTimeString) =>
-      Try {
-        TimeUtil.withDefaults.toZonedDateTime(dateTimeString)
-      }.getOrElse(
+    val startDate = createDateTime(timeConfig.startDateTime)
+    val endDate = createDateTime(timeConfig.endDateTime)
+
+    if (startDate.isAfter(endDate))
+      throw new InvalidConfigParameterException(
+        s"Invalid time configuration." +
+          s"Please ensure that the start time of the simulation is before the end time."
+      )
+  }
+
+  /** Check for valid dateTime string
```

Comment:
Please update the ScalaDoc _and_ also add a hint that an `InvalidConfigParameterException` is thrown if a faulty date string is passed?